packages/nimble-components/src/rich-text-viewer/index.ts

```diff
@@ -11,7 +18,99 @@ declare global {
 /**
  * A nimble styled rich text viewer
  */
-export class RichTextViewer extends FoundationElement {}
+export class RichTextViewer extends FoundationElement {
+    /**
+     * Whether to grow the height of the component to fit the content.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: fit-to-content
+     */
+    @attr({ attribute: 'fit-to-content', mode: 'boolean' })
+    public fitToContent = false;
+
+    /**
+     * @public
+     * @remarks
+     * Accessor: uses the parsing logic to convert the input markdown string to render in the component.
+     */
+    public set markdownValue(value: string) {
```

Comment:
Similar to my feedback on the SLE side, I'd vote for just `markdown` as the accessor and attribute name as I don't think the "value" part adds much useful info.

Reply:
I have updated the name in this PR and created a separate PR for updating it in the spec: https://github.com/ni/nimble/pull/1342.